MusicApp.Identity.Application/AutoMapper/AutoMapperProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using MusicApp.Identity.Application.DTOs;
+using MusicApp.Identity.Domain.Entities;
+
+namespace MusicApp.Identity.Application.AutoMapper;
+
+public class AutoMapperProfile : Profile
```

Comment:
provide class with more concrete name. And do you think, that it is good idea to hash like this?

Reply:
> provide class with more concrete name.

What about UserMapperProfile.cs?

> And do you think, that it is good idea to hash like this?

Yes. I find BCrypt a good function for hashing passwords. I didn't really understand this question.

Reply:
I mean, that you should do it in service method or handler to have more control

Reply:
but for now you can leave it like this